projects/testing-library/tests/render.spec.ts

```diff
@@ -601,3 +602,50 @@ describe('inputs and signals', () => {
     expect(typeTests).toBeTruthy();
   });
 });
+
+describe('README examples', () => {
```

Comment:
I think this can become the new example, since the current one is outdated.
I'll create a new issue for this.

Reply:
I did update the example in the README fwiw

Reply:
Reviewing on the phone interface is hard 😅
Thanks, this looks good!